code/_onclick/other_mobs.dm

```diff
@@ -64,8 +64,8 @@
 		return interact(user)
 	return FALSE
 
-/atom/proc/can_interact(mob/user)
-	if(!user.can_interact_with(src))
+/atom/proc/can_interact(mob/user, require_adjacent_turf = TRUE)
+	if(!user.can_interact_with(src, interaction_flags_atom & INTERACT_ATOM_ALLOW_USER_LOCATION))
```

Comment:
require_adjacent_turf doesn't need to exist anymore does it?

Reply:
Correct. Nice spot